lessons/updated-intro-to-ar-development.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Introduction to Mobile Augmented Reality Development in Unity
+authors:
+- Jacob W. Greene
+date: 2015-01-04
```

Comment:
Let's make sure to update this published date, since this is a new lesson.

Reply:
Done